packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptFieldStyling.ts

```diff
@@ -0,0 +1,28 @@
+import { css } from '@emotion/react';
+
+// import { DANGER, SUCCESS } from "../../../stylesheets/emotion/colors";
```

Comment:
This was my bad before transferring this story it to you: the proper colors from the theme must still be applied in this file.

Reply:
Fixed